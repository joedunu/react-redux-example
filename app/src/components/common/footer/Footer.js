import React from 'react'
import BottomNavigation, { BottomNavigationButton } from 'material-ui/BottomNavigation'
import RestoreIcon from 'material-ui-icons/Restore'
import FavoriteIcon from 'material-ui-icons/Favorite'
import LocationOnIcon from 'material-ui-icons/LocationOn'

const Footer = () => (
  <div>
    <BottomNavigation
      value={0}
      onChange={() => {}}
      showLabels
      className={''}
    >
      <BottomNavigationButton label='Recents' icon={<RestoreIcon />} />
      <BottomNavigationButton label='Favorites' icon={<FavoriteIcon />} />
      <BottomNavigationButton label='Nearby' icon={<LocationOnIcon />} />
    </BottomNavigation>
  </div>
)

export default Footer
