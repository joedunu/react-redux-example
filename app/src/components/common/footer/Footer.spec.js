import React from 'react'
import {shallow} from 'enzyme'
import {shallowToJson} from 'enzyme-to-json'

import Footer from './Footer'

describe('Footer', () => {
  test('should pass snapshot test', () => {
    const component = shallow(<Footer />)
    const tree = shallowToJson(component)
    expect(tree).toMatchSnapshot()
  })
})
