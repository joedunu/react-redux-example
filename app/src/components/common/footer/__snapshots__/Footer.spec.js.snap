// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should pass snapshot test 1`] = `
<div
  className="footer"
>
  Â© React Redux Example
</div>
`;
